android/translate-showcase/app/src/main/java/com/google/mlkit/showcase/translate/analyzer/TextAnalyzer.kt

```diff
@@ -49,17 +49,20 @@ class TextAnalyzer(
     // Flag to skip analyzing new available frames until previous analysis has finished.
     private var isBusy = false
```

Comment:
Now that we are using the newer version of the CameraX API, which has explicit backpressure signaling by closing the `ImageProxy`, I think we don't need this flag anymore, if we set the backpressure strategy to `STRATEGY_KEEP_ONLY_LATEST`.